Source/Core/VideoBackends/Vulkan/VulkanContext.cpp

```diff
@@ -77,22 +79,48 @@ bool VulkanContext::CheckValidationLayerAvailablility()
   res = vkEnumerateInstanceLayerProperties(&layer_count, layer_list.data());
   ASSERT(res == VK_SUCCESS);
 
-  // Check for both VK_EXT_debug_report and VK_LAYER_LUNARG_standard_validation
-  return (std::find_if(extension_list.begin(), extension_list.end(),
-                       [](const auto& it) {
-                         return strcmp(it.extensionName, VK_EXT_DEBUG_REPORT_EXTENSION_NAME) == 0;
-                       }) != extension_list.end() &&
-          std::find_if(layer_list.begin(), layer_list.end(), [](const auto& it) {
-            return strcmp(it.layerName, "VK_LAYER_KHRONOS_validation") == 0;
-          }) != layer_list.end());
+  bool supports_validation_layers =
+      std::find_if(layer_list.begin(), layer_list.end(), [](const auto& it) {
+        return strcmp(it.layerName, VALIDATION_LAYER_NAME.c_str()) == 0;
+      }) != layer_list.end();
+
+  bool supports_debug_utils =
+      std::find_if(extension_list.begin(), extension_list.end(), [](const auto& it) {
+        return strcmp(it.extensionName, VK_EXT_DEBUG_UTILS_EXTENSION_NAME) == 0;
+      }) != extension_list.end();
+
+  if (!supports_debug_utils && supports_validation_layers)
+  {
+    extension_count = 0;
+    res = vkEnumerateInstanceExtensionProperties(VALIDATION_LAYER_NAME.c_str(), &extension_count,
+                                                 nullptr);
+    if (res != VK_SUCCESS)
+    {
+      LOG_VULKAN_ERROR(res, "vkEnumerateInstanceExtensionProperties failed: ");
+      return false;
+    }
+
+    extension_list.resize(extension_count);
+    res = vkEnumerateInstanceExtensionProperties(VALIDATION_LAYER_NAME.c_str(), &extension_count,
+                                                 extension_list.data());
+    ASSERT(res == VK_SUCCESS);
+    supports_debug_utils =
+        std::find_if(extension_list.begin(), extension_list.end(), [](const auto& it) {
+          return strcmp(it.extensionName, VK_EXT_DEBUG_UTILS_EXTENSION_NAME) == 0;
+        }) != extension_list.end();
+  }
+
+  // Check for both VK_EXT_debug_utils and VK_LAYER_LUNARG_standard_validation
+  return supports_debug_utils && supports_validation_layers;
```

Comment:
This comment confuses me. `VK_LAYER_LUNARG_standard_validation` appears in `VulkanContext::CreateDevice`, but `VulkanContext::CheckValidationLayerAvailablility` previously checked `VK_LAYER_KHRONOS_validation` instead, which seems like it's probably different (although I'm not 100% sure of that).

Reply:
It should say `VK_LAYER_KHRONOS_validation`.